libparsec/crates/protocol/tests/invite.rs

```diff
@@ -61,20 +61,6 @@ fn serde_invite_new_req(#[case] raw: &[u8], #[case] expected: authenticated_cmds
 }
 
 #[parsec_test]
-#[case::ok_without_email_sent(
```

Comment:
If this is no longer valid, it would be good to add it as test of invalid data (with a comment explaining this used to be valid in APIv2)

Reply:
Does adding `#[should_panic]` be enough ?

Reply:
I am not sure this is possible with the way the tests are written (or maybe I do not understand what you mean by "test of invalid data" :thinking:)

If I restore that test, the following will not compile because `email_sent` cannot be absent in v3/v4, it needs to be of type `InvitationEmailSentStatus`:
```rust
    authenticated_cmds::invite_new::Rep::Ok {
        token: InvitationToken::from_hex("d864b93ded264aae9ae583fd3d40c45a").unwrap(),
        email_sent: Maybe::Absent, // <-- Compile Error
    }
```

I can use a specific status (such as  `InvitationEmailSentStatus::NOT_AVAILABLE` or `InvitationEmailSentStatus::BAD_RECIPIENT`) but then I wouldn't be testing "invalid data", just a different status value. 

I suppose I can also restore v2 API schemas just to use them in tests, but it seem overkill isn't it?

Reply:
You could add `#[should_panic]` (because with the invalid bytes the `Rep::load(bytes).unwrap()` will panic) 
or you create a new test that does `unwrap_err` instead at the `Rep::load(bytes)`

Reply:
I'm not sure how the `#[should_panic]` would help if the code is not _compiling_ 🫤
From what I understand, I need to specify a concrete a status or use a different `Rep` (because the one coming from `authenticated_cmds::latest::invite_new::Rep` does require an email_sent status)

Reply:
You could use any value for the expected value, since it would fail at the `Rep::load` (would be safer if we used a `expect` with a message)

Reply:
Done